### PR TITLE
Rest client integration test added

### PR DESCRIPTION
**Pull request by irenakezic**
Rest client integration test added as agreed in [#8420](https://github.com/quarkusio/quarkus/pull/8420)

### CODE REVIEW

File: integration-tests/pom.xml [hunk 1]
```diff
@@ -104,6 +104,7 @@
         <module>bootstrap-config</module>
         <module>injectmock</module>
         <module>reactive-messaging-amqp</module>
+        <module>resteasy-client</module>
```
Comment (gsmet) [inline integration-tests/pom.xml:107]: Let's be consistent with the name of the extension: it should be `rest-client`.

File: integration-tests/resteasy-client/pom.xml [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<artifactId>quarkus-integration-tests-parent</artifactId>
+		<groupId>io.quarkus</groupId>
+		<version>999-SNAPSHOT</version>
+		<relativePath>../</relativePath>
+	</parent>
+	<modelVersion>4.0.0</modelVersion>
+	<artifactId>quarkus-integration-tests-resteasy-client</artifactId>
```
Comment (gsmet) [inline integration-tests/resteasy-client/pom.xml:12]:
```suggestion
	<artifactId>quarkus-integration-tests-rest-client</artifactId>
```

File: integration-tests/resteasy-client/pom.xml [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+	<name>Quarkus - Integration Tests - Resteasy client</name>
```
Comment (gsmet) [inline integration-tests/resteasy-client/pom.xml:13]:
```suggestion
	<name>Quarkus - Integration Tests - REST Client</name>
```

File: integration-tests/pom.xml [hunk 4]
```diff
@@ -104,6 +104,7 @@
...
     </modules>
```
Comment (gsmet) [inline integration-tests/pom.xml:108]: Also the module needs to be added here: https://github.com/quarkusio/quarkus/blob/master/.github/workflows/ci-actions.yml#L392 (and please add 10 minutes to the timeout a few lines above)

File: integration-tests/resteasy-client/src/test/java/org/acme/rest/client/multipart/MultipartResourceIT.java [hunk 5]
```diff
@@ -0,0 +1,7 @@
+package org.acme.rest.client.multipart;
```
Comment (gsmet) [inline integration-tests/resteasy-client/src/test/java/org/acme/rest/client/multipart/MultipartResourceIT.java:1]: The package should be consistent with the other classes.

File: integration-tests/resteasy-client/src/test/java/org/acme/rest/client/multipart/MultipartResourceTest.java [hunk 6]
```diff
@@ -0,0 +1,26 @@
+package org.acme.rest.client.multipart;
```
Comment (gsmet) [inline integration-tests/resteasy-client/src/test/java/org/acme/rest/client/multipart/MultipartResourceTest.java:1]: Same here.

**Review by gsmet — CHANGES_REQUESTED** [review]
Cool, thanks!

I added a bunch of comments, mostly minor nitpicking!

**Comment by irenakezic** [comment]
@gsmet pushed another commit with the suggested changes

**Review by gsmet — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**